vcbuild.bat

```diff
@@ -550,7 +557,4 @@ if not "%DISTTYPE%"=="custom" (
   set TAG=%DISTTYPE%%DATESTRING%%COMMIT%
 )
 set FULLVERSION=%NODE_VERSION%-%TAG%
-
-:exit
-if not defined DISTTYPEDIR set DISTTYPEDIR=%DISTTYPE%
```

Comment:
So I removed it because of `:exit` is redefined from L511
Fix PR https://github.com/nodejs/node/pull/13969